### ISSUE TITLE
error: metadata-generation-failed while installing PaddleOCR

### ISSUE THREAD

**Issue by HENNESSYxie** [PaddlePaddle/PaddleOCR#9451]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：Google Colab
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：
- 运行指令/Command Code：!pip install .
- 完整报错/Complete Error Message：
- `Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/
Obtaining file:///content/PaddleOCR
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> See above for output.
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  Preparing metadata (setup.py) ... error
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.`
![image](https://user-images.githubusercontent.com/83642825/225315127-3f2752e0-f3b1-4e9f-ae38-5122a4879e7d.png)

**Comment by HENNESSYxie** [comment]
@tink2123 Can i get some help?

**Comment by tink2123** [comment]
Try to upgrade pip and then manually install dependencies again
```
pip install --upgrade pip
pip install -r requirements.txt
```
requirements.txt can be found here: https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/requirements.txt

**Comment by HENNESSYxie** [comment]
Requirements installing manually is working, but should I run setup.py? In this script also requirements from "ppstructure/recovery/requirements.txt". After installing requirements manually, setup.py gives the same error, which I described before

> Try to upgrade pip and then manually install dependencies again
> 
> ```
> pip install --upgrade pip
> pip install -r requirements.txt
> ```
> 
> requirements.txt can be found here: https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/requirements.txt

Requirements installing manually is working, but should I run setup.py? In this script also requirements from "ppstructure/recovery/requirements.txt". After installing requirements manually, setup.py gives the same error, which I described before
